### PR TITLE
css-adjustments-to-fit-fetched-data-on-landing-page

### PR DESCRIPTION
**Pull request by ibraheemkabir**
## What does this PR does?
Fix CSS issues that arose when the redux implementation is done.
Also it refractor some CSS file to avoid conflicts.

landing-page-css-fix

### CODE REVIEW

File: src/components/carousel/carousel.scss [hunk 1]
```diff
@@ -30,14 +30,29 @@
...
+  p.author {
+    margin: 0 0 0 auto;
+    width: 48%;
+    text-align: left;
+    background-color: transparent;
+    padding-top: 0.3rem;
+    padding-bottom: 0;
+    font-size: 0.9rem;
+    font-weight: 400;
+    font-family: Roboto;
```
Comment (iMichaelOwolabi) [inline src/components/carousel/carousel.scss:43]: Nice Work Kabir. Is it possible to inherit this font-family from the body since it is our global font

File: src/containers/Homecontainer/HomeContainer.scss [hunk 2]
```diff
@@ -86,7 +86,7 @@ img.ui.image.AdvertImg{
   display: block;
 }
 
-.ui {
+.Main .ui {
```
Comment (iAfolayan) [inline src/containers/Homecontainer/HomeContainer.scss]:
```suggestion
.main .ui {
```

File: src/containers/Homecontainer/HomeContainer.scss [hunk 3]
```diff
@@ -197,13 +216,19 @@ img.ui.image.AdvertImg{
     visibility: visible;
     opacity: 1;
   }
+  .ui.feed > .event > .content .summary {
+    height: auto;
+  }
 }
 
 p.Popular.title {
```
Comment (iAfolayan) [inline src/containers/Homecontainer/HomeContainer.scss]:
```suggestion
p.popular.title {
```

File: src/containers/Homecontainer/HomeContainer.scss [hunk 4]
```diff
@@ -331,6 +361,10 @@ a.header {
   border-style: none;
 }
 
+p.descriptionBody {
+  height: 6rem;
+}
+
 @media only screen and (max-width: 1200px) {
   .Main {
```
Comment (iAfolayan) [inline src/containers/Homecontainer/HomeContainer.scss]:
```suggestion
  .main {
```

File: src/components/widgets/Articlewidgets/widget.scss [hunk 5]
```diff
@@ -82,11 +84,27 @@
...
     i.icon {
       padding: 0.7692rem;
       padding-right: 0.3846rem;
-      padding-left: 1.2308rem;
+      padding-left: 0.308rem;
     }
   }
 }
+
+.Icons span {
```
Comment (codrex) [inline src/components/widgets/Articlewidgets/widget.scss]:
```suggestion
.icons span {
```

File: .travis.yml [hunk 6]
```diff
@@ -12,6 +12,6 @@ before_script:
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
 script:
-  - npm test
+  - npm test -- -u
```
Comment (codrex) [inline .travis.yml]: `-- -u` why this change?

File: src/components/widgets/Articlewidgets/test/__snapshots__/widgets.test.js.snap [hunk 7]
```diff
@@ -25,7 +25,7 @@ exports[`button test should test for the button component 2`] = `
   className="widgetContainer"
 >
   <div
-    className="Icons"
+    className=".icons"
```
Comment (codrex) [inline src/components/widgets/Articlewidgets/test/__snapshots__/widgets.test.js.snap]:
```suggestion
    className="icons"
```

**Review by codrex — APPROVED** [review]
